doc/releases/release-notes-4.0.rst

```diff
@@ -405,9 +407,13 @@ Drivers and Sensors
 
 * Serial
 
+  * ESP32: Added UART support for ESP32-C2 and ESP8684.
```

Comment:
Fyi @sylvioalves -- looking at https://github.com/zephyrproject-rtos/zephyr/pull/80912 it looks like you didn't enumerate all the driver updates for ESP32-C2 / ESP8684. I'm fine either way i.e only mention "Added SoC support" as it kind of implies that basic peripherals are supported, or actually list them all, including UART as proposed in this PR, but it needs to be consistent

Reply:
In that case, I will remove my entry and let them do the whole thing.